src/index.ts

```diff
@@ -226,3 +226,23 @@ export { acp_v1 } from "./acp/v1";
  * them will be included in your bundle.
  */
 export { acp_v2 } from "./acp/v2";
+
+/**
+ * This API is still experimental, and subject to change. It builds on top of both
+ * ACP and ACL, aiming at being adaptable to any Access Control system that may be
+ * implemented in Solid. That is why it is purely Resource-centric: the library
+ * discovers metadata associated to the Resource itself, and calls the appropriate
+ * underlying API to deal with the Access Control in place for the target Resource.
+ *
+ * Being still developped, the following export is *only* intended for experimentation
```

Comment:
```suggestion
 * Being still developed, the following export is *only* intended for experimentation
```

though I think the tense is off. Unfortunately I'm not enough of an English scholar to know when terms like "present participle" and all that apply, but I think it should be something like the following :P 

```suggestion
 * As it is still under development, the following export is *only* intended for experimentation
```